components/browser/session/src/main/java/mozilla/components/browser/session/SessionManager.kt

```diff
@@ -260,6 +260,19 @@ class SessionManager(
         }
     }
 
+    /**
+     * Removes a selective list of sessions.
+     */
+    fun removeSelectedSessions(sessionsToRemove: List<Session>) {
+        for (session in sessionsToRemove) {
+            delegate.remove(session)
+        }
+
+        store?.syncDispatch(
+            TabListAction.RemoveTabsAction(sessionsToRemove.map { it.id })
+        )
+    }
```

Comment:
We didn't want to add anything to `SessionManager` anymore. But since this does not really change the behavior and just let's us do a bulk operation in `BrowserStore`, I think this is fine! :)